maven-publish.gradle

```diff
@@ -0,0 +1,73 @@
+apply plugin: "maven-publish"
+apply plugin: "signing"
+
+// Dokka is optional but recommended.
+// Sonatype will not accept a published artifact without at least an empty Javadoc JAR.
+task javadocsJar(type: Jar, dependsOn: "dokkaJavadoc") {
+    getArchiveClassifier().set("javadoc")
+    from dokkaJavadoc.outputDirectory
+}
+
+task sourcesJar(type: Jar) {
+    getArchiveClassifier().set("sources")
+    from sourceSets.main.allJava
+}
+
+afterEvaluate {
+    publishing {
+        // gets specific props defined per each module which breaks the need for multiple(per module) maven-publish files
+        def properties = project.ext.mavenPublishProperties
+
+        repositories {
+            maven {
+                name "SonatypeMavenCentral"
+                url properties.repository.url
+                credentials {
+                    username properties.repository.username
+                    password properties.repository.password
+                }
+            }
+        }
+        publications {
+            release(MavenPublication) {
+                // Only needed for non Android modules. Uncomment if it is needed.
+//                from components.java
+
+                groupId = properties.group
+                version = properties.version
+                artifactId = properties.artifactId
+
+//                artifact bundleReleaseAar
```

Comment:
Maybe a comment about when to uncomment this?

Reply:
Probably makes sense even though it's described in handbook. That's what I meant with the note in description about changing the handbook chapter as well.

Reply:
Since we include this file in all projects, we can't really comment/uncomment things to make it work if project have both Android and non-Android modules.
Let's either check if project has plugin with id `com.android.library`/`com.android.application` or let's introduce two separate files. I prefer if check, do you see any problem with it?

Reply:
@antunflas No I like the idea 👍